### ISSUE TITLE
Dark spot after visiting dungeon

### ISSUE THREAD

**Issue by ZeWoll** [CleverRaven/Cataclysm-DDA#64382]
### Describe the bug

I was wandering around the map when I stumbled upon these dark spots where the dungeons I previously visited had spawned.

### Attach save file

[Za Warudo-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/11010475/Za.Warudo-trimmed.tar.gz)

### Steps to reproduce

I haven't tested much so I'm not sure.
1. Wait for a portal storm I guess and enter the strange location/dungeon
2. Go through the dungeon and exit it with an artifact
3. Return to a safe place
4. After it has passed, revisit the location

### Expected behavior

There should be no dark spot, unless it is something lore related? Which I'm guessing it isn't.

### Screenshots

![image](https://user-images.githubusercontent.com/88143686/226161908-e33e135e-5669-479c-ad76-32817ee73280.png)
![image](https://user-images.githubusercontent.com/88143686/226161918-7a6996fb-454a-4d13-8e80-01fa6d2337c7.png)
![image](https://user-images.githubusercontent.com/88143686/226161938-3c99c0b4-ddec-4ad9-8a00-9af509393113.png)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.2728 (22H2)
- Game Version: 3b9d021 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

**Comment by andrei8l** [comment]
The dungeon wasn't removed. It's supposed to be removed when you exit through the `hole to outside` or the portal storm ends. I've tried a couple of things but I can't reproduce it not reverting. Anything else you can remember about your interaction with the dungeon?

**Comment by ZeWoll** [comment]
Well the portal storm was at night when I went to the dungeon, I also walked through the portal at the end of the dungeon a couple of times as I was testing how it worked but nothing seemed to happen so I eventually left and everything seemed to be okay. Maybe it's related to that? I didn't do much more really, but I also remember using a mining helmet with the light turned on so I could see as I made my way there, through the dungeon and back to the shelter.
